api/src/services/specifications.ts

```diff
@@ -323,6 +323,16 @@ class OASSpecsService implements SpecificationSubService {
 
 		components.schemas = {};
 
+		// Always includes the schemas with these names
+		const includeSchemas = ['Query', 'x-metadata'];
```

Comment:
I think moving this to `/api/constants.ts` would be a more appropriate place.

Reply:
@br41nslug moved to constants.ts :+1:

Reply:
Awesome! 🚀 It might be getting a little nitpicky but im thinking perhaps `OAS_REQUIRED_SCHEMAS` is little more descriptive 🤔

Reply:
No problem, changed 👍🏼